README.md

```diff
@@ -117,7 +117,7 @@ Current version is 3.0. Upgrade guides:
 | quic | Set to `true` to enable QUIC support | bool | `"false"` | no |
 | security\_policy | The resource URL for the security policy to associate with the backend service | string | `"null"` | no |
 | ssl | Set to `true` to enable SSL support, requires variable `ssl_certificates` - a list of self_link certs | bool | `"false"` | no |
-| ssl\_certificates | SSL cert self_link list. Required if `ssl` is `true` and `use_ssl_certificates` is `true`. | list(string) | `<list>` | no |
+| ssl\_certificates | SSL cert self_link list. Required if `ssl` is `true` and no `private_key` and `certificate` is provided. | list(string) | `<list>` | no |
```

Comment:
I don't think this belongs here, but one of `make build` or `make docker_test_lint` seem to have added it.